Spanner/src/Session/Session.php

```diff
@@ -185,7 +185,7 @@ public function expiration()
     public function __debugInfo()
     {
         return [
-            'connection' => get_class($this->connection),
+            'connection' => is_object($this->connection) ? get_class($this->connection) : null,
```

Comment:
Could you tell me again what this does? We don't mind keeping it if it's fixing an error, but since all tests were passing without it, I am assuming this was coming from somewhere else.

Now that we've made a bunch of changes, do you think we can revert this?

Reply:
I believe this was from when I executed all tests from the repo's root, I just tested this again, and it works without the change. I dropped the commit and will look over the other changes as well.